tests/test_integration_real_llm.py

```
@@ -0,0 +1,127 @@
+"""Integration tests with real LLM calls to verify prompt-parser compatibility."""
+
+import os
+import pytest
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)